qtopconf.yaml

```diff
@@ -84,6 +84,7 @@ state_abbreviations:
     C: cancelled_of_user
     E: exiting_of_user
     W: waiting_of_user
+    H: holding_of_user
```

Comment:
OK, I think I know what is the problem with all these states:
* they should be instead `hold_state_of_user` etc; makes more sense for readability esp. w. experienced sysadmins